src/ecu_lua_script.cpp

```diff
@@ -620,3 +650,55 @@ void EcuLuaScript::registerIsoTpSender(IsoTpSender* pSender) noexcept
 {
     pIsoTpSender_ = pSender;
 }
+
+string EcuLuaScript::intToHexString(const uint8_t* buffer, const size_t num_bytes)
+{
+    string a = "";
+
+    for (unsigned int i = 0; i < num_bytes; i++)
```

Comment:
https://stackoverflow.com/questions/26503606/better-way-to-convert-a-vector-of-uint8-to-an-ascii-hexadecimal-string

Reply:
Done